mol_dqn/chemgraph/optimize_logp_of_800_molecules.py

```diff
@@ -135,7 +135,7 @@ def _reward(self):
       # similar to the target at all, but has a high logP. The logP improvement
       # can be 20, and the similarity difference can be 0.5. To discourage that
       # molecule, similarity difference is timed by 20 / 0.5 = 40.
```

Comment:
Please update this comment to match the code.

Reply:
Done